libs.versions.toml

```diff
@@ -0,0 +1,161 @@
+# Also see settings.gradle file for dynamic versionCatalogs dependencies
+
+[versions]
+antlrVersion = "4.8"
+awsSdkVersion = "2.20.18"
+flink115Version = "1.15.0"
+flink116Version = "1.16.2"
+flink117Version = "1.17.1"
+hadoop2Version = "2.7.3"
+hive2Version = "2.3.9"
+hive3Version = "3.1.3"
+jackson211Version = "2.11.4"
+jackson212Version = "2.12.3"
+jackson213Version = "2.13.4"
+jacksonVersion = "2.14.2"
+jettyVersion = "9.4.43.v20210629"
+junitJupiterVersion = "5.9.2"
+mockitoVersion = "4.11.0"
+nessieVersion = "0.60.1"
+orcVersion = "1.8.3"
+parquetVersion = "1.13.1"
+springBootVersion = "2.5.4"
+tez010Version = "0.10.2"
+tez08Version = "0.8.4"
+
+[libraries]
+activation = "javax.activation:activation:1.1.1"
+aircompressor = "io.airlift:aircompressor:0.24"
+aliyun-sdk-oss = "com.aliyun.oss:aliyun-sdk-oss:3.10.2"
+antlr-antlr4 = { module = "org.antlr:antlr4", version.ref = "antlrVersion" }
+antlr-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlrVersion" }
+arrow-memory-netty = "org.apache.arrow:arrow-memory-netty:11.0.0"
+arrow-vector = "org.apache.arrow:arrow-vector:11.0.0"
+avro-avro = "org.apache.avro:avro:1.11.1"
+awssdk-apache-client = { module = "software.amazon.awssdk:apache-client", version.ref = "awsSdkVersion" }
+awssdk-auth = { module = "software.amazon.awssdk:auth", version.ref = "awsSdkVersion" }
+awssdk-dynamodb = { module = "software.amazon.awssdk:dynamodb", version.ref = "awsSdkVersion" }
+awssdk-glue = { module = "software.amazon.awssdk:glue", version.ref = "awsSdkVersion" }
+awssdk-kms = { module = "software.amazon.awssdk:kms", version.ref = "awsSdkVersion" }
+awssdk-lakeformation = { module = "software.amazon.awssdk:lakeformation", version.ref = "awsSdkVersion" }
+awssdk-s3 = { module = "software.amazon.awssdk:s3", version.ref = "awsSdkVersion" }
+awssdk-sts = { module = "software.amazon.awssdk:sts", version.ref = "awsSdkVersion" }
+awssdk-url-connection-client = { module = "software.amazon.awssdk:url-connection-client", version.ref = "awsSdkVersion" }
+caffeine = "com.github.ben-manes.caffeine:caffeine:2.9.3"
+calcite-core = "org.apache.calcite:calcite-core:1.10.0"
+calcite-druid = "org.apache.calcite:calcite-druid:1.10.0"
+errorprone-annotations = "com.google.errorprone:error_prone_annotations:2.3.3"
+findbugs-jsr305 = "com.google.code.findbugs:jsr305:3.0.2"
+flink115-avro = { module = "org.apache.flink:flink-avro", version.ref = "flink115Version" }
```

Comment:
(Same comment as for the Jackson dependencies below, including the "BOM approach")

Reply:
I did transition to usage of rich version for Flink but I can't find bom for Flink.